src/client/components/autocomplete.vue

```diff
@@ -2,7 +2,7 @@
 <div class="swhvrteh" @contextmenu.prevent="() => {}">
 	<ol class="users" ref="suggests" v-if="type === 'user'">
 		<li v-for="user in users" @click="complete(type, user)" @keydown="onKeydown" tabindex="-1" class="user">
-			<img class="avatar" :src="user.avatarUrl"/>
+			<img class="avatar" :src="user.avatarUrl" :alt="image.name"/>
```

Comment:
I think `image.name` is file name of image (it is maybe "Screen Shot - 20200730012345.png", "IMG_1234.jpg" or other), it is not alt text.
in this context, I think we must use "title" attribute instead of "alt" attribute.

do you think about this?

Reply:
I'm not sure if user avatars have alt text in the first place. I would assume most do not.